demo/index.html

```diff
@@ -17,25 +17,22 @@
     <meta name="mobile-web-app-capable" content="yes">
     <meta name="apple-mobile-web-app-capable" content="yes">
 
-    <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+    <script src="https://unpkg.com/@webcomponents/webcomponentsjs@^2.0.0/webcomponents-loader.js"></script>
 
-    <script type="module" src="../../polymer/polymer-legacy.js"></script>
-    <script type="module" src="../../iron-demo-helpers/demo-pages-shared-styles.js"></script>
-    <script type="module" src="../../iron-demo-helpers/demo-snippet.js"></script>
-    <script type="module" src="../iron-image.js"></script>
-
-    <!-- FIXME(polymer-modulizer):
-        These imperative modules that innerHTML your HTML are
-        a hacky way to be sure that any mixins in included style
-        modules are ready before any elements that reference them are
-        instantiated, otherwise the CSS @apply mixin polyfill won't be
-        able to expand the underlying CSS custom properties.
-        See: https://github.com/Polymer/polymer-modulizer/issues/154
-        -->
     <script type="module">
-const $_documentContainer = document.createElement('template');
+      import '@polymer/iron-demo-helpers/demo-pages-shared-styles.js';
+      import '@polymer/iron-demo-helpers/demo-snippet.js';
+      import '@polymer/polymer/polymer-legacy.js';
+      import '../iron-image.js';
+
+      window.load = function(id) {
+        document.getElementById(id).src = './polymer.svg?' + Math.random();
+      };
+    </script>
 
-$_documentContainer.innerHTML = `<custom-style>
+  </head>
+  <body unresolved>
+    <custom-style>
       <style is="custom-style" include="demo-pages-shared-styles">
```

Comment:
So moving `<style is="custom-style" include="demo-pages-shared-styles">` to `<body>` works? Wished I knew that before keeping template stamping in JS in other elements 😦

Reply:
I'm actually not sure why it works here, none of the others I've done before this have. :/